.azure_pipelines/olive-aml-ci.yaml

```diff
@@ -0,0 +1,57 @@
+trigger:
+  batch: true
+  branches:
+    include:
+    - main
+  paths:
+    exclude:
+    - docs/*
+    - examples/*
+    - README.md
+    - CONTRIBUTING.md
+    - LICENSE
+pr:
+  branches:
+    include:
+    - main
+  paths:
+    include:
+    - olive/azureml/*
+    - olive/systems/azureml/*
+
+jobs:
+# Linux examples test
+- template: job_templates/olive-example-template.yaml
+  parameters:
+    name: Linux_CI
```

Comment:
nit: name include aml so that we know which pipeline it belongs to in the github status

Reply:
Nevermind, github displays the pipeline name before the job name